src/System.Globalization/tests/CultureInfo/CultureInfoDateTimeFormat.cs

```diff
@@ -37,6 +38,44 @@ public void PosTest2()
             Assert.Equal("HH|mm", myCultureInfo.DateTimeFormat.ShortTimePattern);
         }
 
+        [Theory]
+        [InlineData("en-US", "h:mm tt")]
+        public void TestShortTimePatternLocale(string locale, string expected)
+        {
+            CultureInfo myTestCulture = new CultureInfo(locale);
+            DateTimeFormatInfo dtf = myTestCulture.DateTimeFormat;
+            Assert.Equal(expected, dtf.ShortTimePattern);
+        }
+
+        [InlineData("fi-FI", "H:mm", "H.mm")]
```

Comment:
I would be fine keeping it as is, so we could easily add additional locale cases (the difference between this and the next one is just having a way to signal that ICU data differs)